docker/Dockerfile_base

```diff
@@ -13,6 +13,9 @@ RUN curl -fsSL https://packages.redis.io/gpg | gpg --dearmor -o /usr/share/keyri
 # ALPINE DISTRO UPDATES
 # RUN apk update && apk add --update autoconf automake bash build-base coreutils curl git libxml2 libxml2-dev libxslt libxslt-dev nodejs openssh openssl openssl-dev perl ruby-nokogiri sqlite sqlite-dev tar shared-mime-info protobuf tzdata redis
 
+# Add dependency for FFI/AgentLib gem support.
+RUN apt-get install musl-dev -y && ln -s /usr/lib/x86_64-linux-musl/libc.so /lib/libc.musl-x86_64.so.1
+
```

Comment:
Just adding to the base doesn't seem to work. still receiving this error:

```
rake aborted!
LoadError: Could not open library '/usr/local/bundle/gems/contrast-agent-lib-0.1.0/lib/../usage_files/libcontrast_c.so': libc.musl-x86_64.so.1: cannot open shared object file: No such file or directory
/usr/local/bundle/gems/ffi-1.15.5/lib/ffi/library.rb:145:in `block in ffi_lib'
```
So I'll just put things as they were.

Reply:
If you're trying it locally, you'll need to remove the intermediate instances for base and agent to get the change in the final docker image. 

I don't want to hold this up, since I still hope that we can roll this back eventually, so approving for now.